test/docker/integration.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+    fcm-mock:
+        image: mongooseim/fcm-mock-server
+        container_name:  fcm-mock
+        ports:
+            - "4001:4001"
```

Comment:
I don't remember which is HTTPS, but the HTTPS is not needed here, as pretty much only mpush should connect to it - and since mpush is within docker-compose network, binding locally is superfluous.

Reply:
Apparently it fails to connect if we do not expose both